src/components/common/Button.js

```diff
@@ -1,37 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import {
-  Text,
-  TouchableOpacity,
-  StyleSheet,
-} from 'react-native';
+import { Text, TouchableOpacity, StyleSheet } from 'react-native';
 import { useTheme } from '@react-navigation/native';
 import TextStyles from 'helpers/TextStyles';
 
+const styles = StyleSheet.create({
+  button: {
+    alignSelf: 'stretch',
+    alignItems: 'center',
+    borderWidth: 1,
+    marginTop: 10,
+    padding: 5,
+    borderRadius: 5,
+  },
+});
+
 const Button = props => {
+  const { style, textStyle, title } = props;
```

Comment:
In the Login component you destructured props in the declaration, is it a convention that when 1 thing is extracted it goes there and when more than one you place it in a new line inside?

Reply:
No, indeed I prefer to extract always in the declaration but in this case isn't (without changing de behavior) since where are spreading the prop to the inner TouchableOpacity. Maybe we should replace `props` with `{ title, textStyle, style, ...rest }` and then spread `rest` to the TouchableOpacity. Thoughts?

Reply:
TBH I don't know what's better. I'm good with whatever 😬